### PR TITLE
Merge month May's CHANGELOG to main branch

### PR DESCRIPTION
**Pull request by mssfang**
added the released version CHANGLOG content back to main repo (which is the missed step should be done at last patch release)

### CODE REVIEW

File: sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,16 @@
 ### Other Changes
 
 
+## 1.1.29 (2024-05-28)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Upgraded `azure-mixedreality-authentication` from `1.2.22` to version `1.2.23`.
```
Comment (vcolin7) [inline sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md:20]: Shouldn't this be `1.2.24`? That's the version of MR Auth we just released.

File: sdk/schemaregistry/azure-data-schemaregistry-apacheavro/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,16 @@
...
 
 
+## 1.1.17 (2024-05-28)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Upgraded `azure-core` from `1.48.0` to version `1.49.0`.
+- Upgraded `azure-data-schemaregistry` from `1.4.5` to version `1.5.0-beta.1`.
```
Comment (vcolin7) [inline sdk/schemaregistry/azure-data-schemaregistry-apacheavro/CHANGELOG.md:21]: This should mention a stable version like `1.4.6` or whatever was recently released for Schema Registry.

**Review by vcolin7 — COMMENTED** [review]
Most things look good. Just added a couple of comments.

File: sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md (hunk 1 above)
Comment (mssfang) [reply sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md:20]: Yeah. Need to fix script to do a better job for the same date release for parent-children dependency packages.

**Review by vcolin7 — APPROVED** [review]

**Pull request merged**